File: Conversor/Conversor/ViewController.m
//
//  ViewController.m
//  Conversor
//
//  Created by René Zelada on 7/7/21.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}
- (IBAction)Convertir:(UIButton *)sender {
    float dolarValue = [self.dinero.text floatValue];
    float res = dolarValue * 8;
    [self.resultado setText: [NSString stringWithFormat:@"Quetzales %.2f",res]];
    self.resultado.hidden = NO;
}


@end

File: HolaUIKIT/HolaUIKIT/ViewController.swift
//
//  ViewController.swift
//  HolaUIKIT
//
//  Created by René Zelada on 6/9/21.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    @IBAction func Boton(_ sender: UIButton) {
        let alert = UIAlertController(title: "Titulo", message: "Msj de la alerta", preferredStyle: .actionSheet)
        let ok = UIAlertAction(title: "Aceptar", style: .default) { (_) in
            print("Aceptar")
        }
        alert.addAction(ok)
        present(alert, animated: true, completion: nil)
    }
    
}

